### ISSUE TITLE
Multiple themes: update how patterns are loaded

### ISSUE THREAD

**Issue by mikachan** [Automattic/themes#5803]
We should update how all patterns are included in our block themes, to allow them to be automatically loaded from Gutenberg, as we did with Archeo here: https://github.com/Automattic/themes/pull/5715

- [x] Remote https://github.com/Automattic/themes/pull/5829
- [x] Livro https://github.com/Automattic/themes/pull/5879
- [x] Stewart https://github.com/Automattic/themes/pull/5854
- [x] Skatepark https://github.com/Automattic/themes/pull/5858

**Comment by MaggieCabrera** [comment]
I added a list of themes so we can use this as an overview issue. Should we do this to Blockbase themes too?

**Comment by mikachan** [comment]
I guess if we include Blockbase themes then we'd maybe need to bump the required WP version for Blockbase to 6.0, as it looks like this is being backported to 6.0 (https://github.com/WordPress/wordpress-develop/pull/2488). We've only just updated the required WP version here https://github.com/Automattic/themes/pull/5823 to 5.8.

Hmm, so I think we're better leaving Blockbase as it is for now, but it would be good if we did these changes to Blockbase themes once WP 6.0 has been released.
